tasks/eth/014-fjord-gas-config/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+OWNER_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
```

Comment:
If these are the same, can we just delete `FOUNDATION_SAFE` for clarity? It seems we only use it in the solidity script, which we can change to reference the `OWNER_SAFE`. Or is one supposed to be different than the other?

Reply:
Will accept a fix commit for this in the next 45 minutes.

Reply:
Done in 7b258dc

Reply:
awesome ty